### PR TITLE
chore: disable kotlin tests/releases

### PR DESCRIPTION
**Pull request by worstell**
don't build kotlin when running just build-all

follow up from #1364 with stuff that was missed

### CODE REVIEW

**Comment by alecthomas** [comment]
Can we also disable Kotlin builds as part of CI and the build process? ie. so that `ftl dev` doesn't try to build it?

File: Justfile [hunk 1]
```diff
@@ -87,9 +87,15 @@ publish-extension: package-extension
...
+#  @mk {{KT_RUNTIME_OUT}} : kotlin-runtime/ftl-runtime -- mvn -f kotlin-runtime/ftl-runtime -Dmaven.test.skip=true -B install
+#  @mk {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}} : {{KT_RUNTIME_OUT}} -- "mkdir -p $(dirname {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}) && install -m 0600 {{KT_RUNTIME_OUT}} {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}"
+#  @mk {{RUNNER_TEMPLATE_ZIP}} : {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}} -- "cd build/template && zip -q --symlinks -r ../../{{RUNNER_TEMPLATE_ZIP}} ."
+
+# Kotlin runtime is temporarily disabled; these instructions create a dummy zip in place of the kotlin runtime jar for
+# the runner.
 build-kt-runtime:
-  @mk {{KT_RUNTIME_OUT}} : kotlin-runtime/ftl-runtime -- mvn -f kotlin-runtime/ftl-runtime -Dmaven.test.skip=true -B install
-  @mk {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}} : {{KT_RUNTIME_OUT}} -- "mkdir -p $(dirname {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}) && install -m 0600 {{KT_RUNTIME_OUT}} {{KT_RUNTIME_RUNNER_TEMPLATE_OUT}}"
+  @mkdir -p build/template/ftl && touch build/template/ftl/temp.txt
```
Comment (worstell) [inline Justfile:98]: cc @alecthomas wdyt? this preserves the mechanism currently in place for deploying runtime-specific data to the runner— we expect an archive formerly just containing the kotlin JAR. now we have now no JAR, so we'll just archive and deploy this empty file? 🤷‍♀️

**Comment by worstell** [comment]
> Can we also disable Kotlin builds as part of CI and the build process? ie. so that `ftl dev` doesn't try to build it?

I think this should be accomplished now, between this and the [other PR](https://github.com/TBD54566975/ftl/pull/1364)—lmk if there's something I missed

File: Justfile (hunk 1 above)
Comment (wesbillman) [reply Justfile:98]: @worstell do you want to comment out these lines as well as you did above for the `build-kt-runtime`? I assume we'll be turning this all back on at some point. Or we can delete and use git for both. whatcha think?

**Review by wesbillman — APPROVED** [review]

File: Justfile (hunk 1 above)
Comment (alecthomas) [reply Justfile:98]: Good idea, works for me.

**Review by alecthomas — APPROVED** [review]

**Comment by alecthomas** [comment]
Thanks Lizzy.

File: Justfile (hunk 1 above)
Comment (worstell) [reply Justfile:98]: maybe i'll just delete for now? it seems like commenting doesn't work from within the directive

**Pull request merged**